graphx/src/main/scala/org/apache/spark/graphx/VertexRDD.scala

```diff
@@ -128,7 +128,7 @@ abstract class VertexRDD[VD](
    *
    * @param other the other RDD[(VertexId, VD)] with which to diff against.
    */
-  def diff(other: RDD[(VertexId, VD)]): VertexRDD[VD]
+  def diff(other: RDD[(VertexId, VD)]): VertexRDD[VD] = ???
```

Comment:
The previous version was an abstract method, which mima complains about (since it breaks source code that extends this class). The new version has an implementation of the method that throws a "NotImplemented" exception (don't recall the exact type).